swedish-oidc-claims-specification.md

```diff
@@ -241,9 +239,9 @@ may be the, but is not required to be, the "personal-id" part of the claim.
 <a name="authentication-information-claims"></a>
 ### 2.3. Authentication Information Claims
 
-An "authentication information" claim delivers information about a specific authentication event. An identity provider SHOULD
-deliver these kinds of claims in an ID token and not from the UserInfo endpoint since the values the claims represent refers
-to an event, and not user properties as such. 
+An "authentication information" claim delivers information about a specific authentication event.
+An identity provider SHOULD deliver these kinds of claims in an ID token and not from the UserInfo
+endpoint since the values the claims represent refers to an event, and not user properties as such. 
 
```

Comment:
OpenID provider instead of identity provider?

Replace "these kinds of claims" with "claims defined in this section" ?